source/chapters/user_interface.rst

```diff
@@ -871,8 +871,20 @@ Equalizers and Gain Knobs
    EQ Controls
 
 **Gain Knob**
+  Gain or Loss in sounds is also referred to as volume levels.
   The gain knob allows you to adjust the gain applied to the deck.
   Use this to compensate for the differences in recording levels between tracks.
+  There are 4 Gain Knobs:
+      1. The Super-Knob
+      2. The Low Filter Knob
+      3. The Mid Filter Knob
+      4. The High Filter Knob
```

Comment:
```suggestion
  There are 4 knobs:
      1. Gain
      2. High filter
      3. Mid filter
      4. Low filter
```
This was wrong.
Also when referencing screenshots please describe elements in the same order the appear in the image.